packages/client/src/components/footer.tsx

```
@@ -0,0 +1,81 @@
+import { FaDiscord , FaLinkedin , FaXTwitter } from 'react-icons/fa6';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)